pkgs/by-name/dq/dqlite/package.nix

```
@@ -22,6 +23,13 @@ stdenv.mkDerivation rec {
     hash = "sha256-7ou077ozbpH21PcvEEcprr4UYJ/X398Ph9dh5C3YyBQ=";
   };
 
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/canonical/dqlite/commit/be453628ce782167f6652c055e600908e2641da7.patch";
```

```suggestion
      url = "https://github.com/canonical/dqlite/commit/be453628ce782167f6652c055e600908e2641da7.patch?full_index=1";
```

Always use it, as GitHub has a habit of  using truncated index hash which leads to hash mismatches later on when the repo gets bigger in terms of history.

Note, this and the above fetchpatch2 change will change the hash so you'll need to regenerate it.